packages/web/src/components/nav/desktop/NowPlayingArtworkTile.tsx

```diff
@@ -170,26 +180,29 @@ export const NowPlayingArtworkTile = () => {
     )
   }
 
-  if (isStreamGated) {
-    return (
-      <div className={styles.root}>
-        {renderDogEar()}
-        {renderCoverArt()}
-      </div>
-    )
-  }
+  const content = (
+    <Box
```

Comment:
heads up we do have mr, ml, mv so could be 

```
mt='unit5' mh='auto' mb={0}
```

Reply:
`auto` is not a valid value for margin. But I can add it if you're cool with it?

Reply:
Oh I see we updated some of the properties to allow plain CSS values but not all. I implemented the rest of them.